tasks/libs/ciproviders/gitlab_api.py

```diff
@@ -1300,6 +1301,18 @@ def update_test_infra_def(file_path, image_tag, is_dev_image=False, prefix_comme
         yaml.dump(test_infra_def, test_infra_version_file, explicit_start=True)
 
 
+def get_test_infra_def_version():
+    """
+    Get TEST_INFRA_DEFINITIONS_BUILDIMAGES from `.gitlab/common/test_infra_version.yml` file
+    """
+    try:
+        with open(Path(".gitlab/common/test_infra_version.yml")) as test_infra_version_file:
+            test_infra_def = yaml.safe_load(test_infra_version_file)
+            return test_infra_def["variables"]["TEST_INFRA_DEFINITIONS_BUILDIMAGES"]
```

Comment:
nit: the `Path` is unnecessary here, to make it useful you could do:

```suggestion
        version_file = Path.cwd() / ".gitlab" / "common" / "test_infra_version.yml"
        test_infra_def = yaml.safe_loads(version_file.read_text(encoding="utf-8"))
        return test_infra_def["variables"]["TEST_INFRA_DEFINITIONS_BUILDIMAGES"]
```